app/Config/Migrations.php

```diff
@@ -44,7 +44,10 @@ class Migrations extends BaseConfig
      * using the CLI command:
      *   > php spark migrate:create
```

Comment:
```suggestion
     *   > php spark make:migration
```
Command `migrate:create` is getting obsolete, better, suggest new command.